### DISCUSSION TITLE
Add-on keeps adding more and more entities

### DISCUSSION THREAD

**Discussion by jjotto** [hass-emulated-hue/core#336] [Q&A] [answered]
I'm not sure what is happening or how really, but the add-on keeps adding more and more entities/devices.  They all seem to be repeats of actual entities/devices.  After I enable it, it shows up on the integrations screen.  I can see the numbers increasing.  I've tried to remove it and add it back, but the numbers keep going up and up.  I disabled the add-on at around 1200 devices/entities.  I guess it is looping over something, but I'm not sure what.  I looked into issues and discussions and didn't see anything like this.  If anyone has any insight, please let me know.  Let me know what information is needed, and I can provide it.

Thanks!

**Comment by alexyao2015** [comment, marked as answer]
Do not add the addon as an integration

**Reply by jjotto** [reply]
Ah, ok, thanks!
